package.json

```diff
@@ -5,6 +5,8 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "docker-build": "docker build -t content-service .",
+    "docker-run": "sh -c \"docker run -it --rm --name content-service-running -p 80:8080 -e RACKSPACE_USERNAME=$RACKSPACE_USERNAME -e RACKSPACE_APIKEY=$RACKSPACE_APIKEY -e RACKSPACE_REGION=$RACKSPACE_REGION -e RACKSPACE_CONTAINER=$RACKSPACE_CONTAINER -e CONTENT_LOG_LEVEL=$CONTENT_LOG_LEVEL content-service\"",
```

Comment:
Not very. This is for you while you're developing the container itself. The production deployment will be done with Ansible (unless I get vetoed on that I guess?). We can make it nicer for other people as other people actually need it to be nicer.